operations/network/management_network/1.2_to_1.3_upgrade.md

```diff
@@ -0,0 +1,163 @@
+  - [Prerequisites](#prerequisites)
+  - [Backup Switch Configurations](#backup-switch-configurations)
+  - [Mellanox](#mellanox)
+  - [Dell](#dell)
+  - [Aruba Spine](#aruba-spine)
+  - [Aruba Leaf and Leaf BMC](#aruba-leaf-and-leaf-bmc)
+
+## Prerequisites
+
+- System is currently running CANU-generated CSM 1.2 configurations.
+- Switch configurations have been generated for CSM 1.3.
+  - [Generate Switch Configurations](generate_switch_configs.md)
+  - Ensure that all site [custom configurations](https://github.com/Cray-HPE/canu/#generate-switch-configs-including-custom-configurations) have been generated.
+- CANU installed with version 1.6.7 or greater.
+  - Run `canu --version` to see version.
+  - Update the CANU RPM from the release tarball. For more information, see [Update CANU From CSM Tarball](canu/update_canu_from_csm_tarball.md)
+
+For this procedure, log in to `ncn-m001` as an administrative user.  The login should be over the site connection (`lan0`), not the `CMN`, to prevent loss of connectivity to the system.
+
+## Backup Switch Configurations
+
+- (`ncn-m001#`) Use CANU to backup the switch running configurations.  Enter the switch administrative password when prompted.
+
+```bash
+mkdir switch-upgrades-csm-1.3
+cd switch-upgrades-csm-1.3
+canu backup network --folder 1.2
+```
+
+- Stage the CANU generated CSM 1.3 switch configurations in the `switch-upgrades-csm-1.3` directory in a subdirectory named 1.3.
+
+## Mellanox
+
+Use CANU Validate to see the differences between the 1.2 and 1.3 switch configurations.
+
+```bash
+canu validate switch config --running ./1.2/sw-spine-002.cfg --generated ./1.3/sw-spine-002.cfg --vendor mellanox
+```
+
+- Take a close look at the output of this, make sure that all the changes are understood.  Changing or removing the following configurations comes with the risk of network interruption:
+  - Generating switch configuration without using `--custom-configuration` to preserve site-specific values.
+  - Changes to MAGP configurations (ISL).
+  - Changes to Spanning Tree.
+  - Changes to ACLs or ACL ordering.
+  - Changes to `VRFs`.
+  - Changes to default route.
+  - Changes to MLAG/LACP.
+- CANU remediation configurations are not supported on Mellanox.  Applied switch changes must be completed from the configuration differences.
+
+- Save this configuration to a new configuration file with the current `CSM` and `CANU` versions.
+
+- (`m001#`) Example:
+
+```text
+sw-spine-001 [mlag-domain: master] (config) # show banner
+
+Banners:
+  Message of the Day (MOTD):
+
+    ###############################################################################
+    # CSM version:  1.2
+    # CANU version: 1.6.5
+    ###############################################################################
+
+  Login:
+    NVIDIA Onyx Switch Management
+
+  Logout:
+
+sw-spine-001 [mlag-domain: master] (config) # configuration write to csm1.3-canu1.6.5
+```
```

Comment:
Indent this block:
````suggestion
  ```text
  sw-spine-001 [mlag-domain: master] (config) # show banner

  Banners:
    Message of the Day (MOTD):

      ###############################################################################
      # CSM version:  1.2
      # CANU version: 1.6.5
      ###############################################################################

    Login:
      NVIDIA Onyx Switch Management

    Logout:

  sw-spine-001 [mlag-domain: master] (config) # configuration write to csm1.3-canu1.6.5
  ```
````